packages/openid4vc/src/openid4vc-verifier/repository/OpenId4VcVerificationSessionRecord.ts

```diff
@@ -0,0 +1,118 @@
+import type { OpenId4VcSiopAuthorizationResponsePayload } from '../../shared/models'
+import type { OpenId4VcVerificationSessionState } from '../OpenId4VcVerificationSessionState'
+import type { RecordTags, TagsBase } from '@credo-ts/core'
+
+import { Jwt, CredoError, BaseRecord, utils } from '@credo-ts/core'
+
+export type OpenId4VcVerificationSessionRecordTags = RecordTags<OpenId4VcVerificationSessionRecord>
+
+export type DefaultOpenId4VcVerificationSessionRecordTags = {
+  verifierId: string
+  state: OpenId4VcVerificationSessionState
+  nonce: string
```

Comment:
The OpenID4VCI related code contains a prop called `cNonce` and here `nonce` is used. The difference isn't very obvious without diving into the specs. Some clarifying comments may help.